templates/items/parts/item-description.hbs

```diff
@@ -63,6 +63,7 @@
             {{#if editable}}
                 <h3 class="description-header">
                     {{localize "DND5E.Description"}}
+                    <i class="accordion-indicator fa-solid fa-chevron-down"></i>
```

Comment:
It could be down to personal experience but I feel like these indicators are typically prefix the title rather than postfix it. For example, the `<details>` tag renders with the indicator first.